### ISSUE TITLE
Black bug when opening files in register

### ISSUE THREAD

**Issue by guilhermepovala** [BIC-MNI/Register#23]
Hi,

Always when I open register with more than one file, this bug occurs. Can someone help me with that?

Thank you!

![2020-01-15_14-27](https://user-images.githubusercontent.com/3400377/72456339-42e93c80-37a3-11ea-899b-1357ec1b70a9.png)

**Comment by vfonov** [comment]
Are you running it over SSH by any chance? Or inside of virtual machine.

**Comment by guilhermepovala** [comment]
No, I'm using in native ubuntu 19.04. I've already tested in ubuntu 16.04 and 18.04.

**Comment by gdevenyi** [comment]
Modern versions of Ubuntu seem to have some bad interactions with nvidia/opengl. Resize the window usually fixes it.

This is probably a bug in GLUT.

**Comment by guilhermepovala** [comment]
I'm using the register that comes with minc. This bug occurs in all computers here in the lab.
When the window is resized the black portion becomes normal. But when another interaction with the images is made, the bug comes back.

I thought it could be some missing library, but it seems to be another problem.

**Comment by vfonov** [comment]
does it have NVIDIA GPU?

**Comment by guilhermepovala** [comment]
Until last week, it doesn't. I installed one yesterday. The bug was happening before and continues after the installation.

**Comment by vfonov** [comment]
does it use NVIDIA driver ?

**Comment by guilhermepovala** [comment]
Now, yes.
![image](https://user-images.githubusercontent.com/3400377/72461209-42ee3a00-37ad-11ea-9443-c65471e907f3.png)

**Comment by vfonov** [comment]
it doesn't say which driver it is using

**Comment by vfonov** [comment]
glxinfo  shows this information

**Comment by guilhermepovala** [comment]
Here is my glxinfo output:

[glxinfo.txt](https://github.com/BIC-MNI/Register/files/4066792/glxinfo.txt)

**Comment by guilhermepovala** [comment]
$ glxinfo | grep OpenGL

![image](https://user-images.githubusercontent.com/3400377/72461743-64035a80-37ae-11ea-8d08-967dba5a01b0.png)

**Comment by vfonov** [comment]
well, it's a mystery.

**Comment by guilhermepovala** [comment]
Is there any chance it can work using docker?

**Comment by gdevenyi** [comment]
I have the same bug in 18.04 with intel:
![Screenshot_20200115_135513](https://user-images.githubusercontent.com/3001850/72462442-1501f900-379f-11ea-90e0-f07afd971dca.png)

**Comment by gdevenyi** [comment]
docker with any graphical application is pretty much a no-go, its a mess to feedthrough the X access.

Plus, I have expectation that docker will fix anything.

**Comment by vfonov** [comment]
we still don't know which version of minc/register that is

**Comment by guilhermepovala** [comment]
Minc 1.9.17
Register 1.5.0 (built Jul  1 2019) git:HEAD/a391751

**Comment by vfonov** [comment]
maybe it's a sign to switch to nifti?

**Comment by vfonov** [comment]
you can try
`LIBGL_ALWAYS_INDIRECT=1 register file1.mnc file2.mnc`

**Comment by guilhermepovala** [comment]
Well, using `LIBGL_ALWAYS_INDIRECT=1` didn't fix the issue.

Some information that might be helpful: 
- When opening a .mnc file as the first one, the bug always happen;
- When a .nii file is the first one, even followed by other .mnc files, it seems that register work normally.

**Comment by vfonov** [comment]
looks like nifti is the answer
